src/screens/TopicMemberScreen/index.js

```diff
@@ -217,8 +170,7 @@ const TopicMemberScreen = () => {
         setIsFollow={setIsFollow}
         isFollow={isFollow}
         getTopicDetail={getTopicDetail}
-      />
-      <Search
+        hasSearch={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `hasSearch` prop has been added to the `NavHeader` component. Ensure that this prop is properly handled in the `NavHeader` component.
```diff
+        hasSearch={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->